README.md

```diff
@@ -119,11 +119,11 @@ type Mutation
 In the above example, the `set_user_name` field will be made available on every `User` object, under the key `set_user_name` (this will be customizable.) So, one could write a resolver:
 
 ```js
-export const UpdateUserNameButton = iso`
+export const UpdateUserNameButton = iso(`
   User.UpdateUserNameButton {
     set_user_name,
   }
-`(({ data: { set_user_name } }) => {
+`)(({ data: { __set_user_name } }) => {
```

Comment:
this should be `set_user_name` after rebasing, magic mutation fields no longer have `__`.